extensions/amp-video/validator-amp-video.protoascii

```diff
@@ -132,6 +132,30 @@ tags: {  # <amp-video> not in amp-story.
   }
 }
 
+tags: {  # <amp-video> in amp-story-page-attachment (same rules as regular AMPHTML).
```

Comment:
@Gregable is there a way for us to actually reuse the same spec?

Reply:
I don't think so, the mandatory ancestor thing is different between them. We have some variants within a spec, but I don't think there is any way to do so w mandatory ancestor.